.env

```diff
@@ -42,5 +46,10 @@ DATAWELL_VENDOR_PROFILE='opac'
 ###< custom ###
 
 ###> enqueue/redis ###
-ENQUEUE_DSN=redis:
+ENQUEUE_DSN=redis://redis:6379
 ###< enqueue/redis ###
+
+###> ES ###
+ESASTICA_HOST=elasticsearch
```

Comment:
`ESASTICA_HOST` -> `ELASTIC_HOST`?

Reply:
The bundle that brings ES to the app is called esatica. But will rename.

Reply:
Fixed